exp-player/addon/components/exp-frame-base.js

```diff
@@ -64,19 +64,16 @@ export default Ember.Component.extend({
             this.set('id', `${kind}-${frameIndex}`);
         }
     },
-    setupParams(params, clean) {
+    setupParams(clean) {
         // Add config properties and data to be serialized as instance parameters (overriding with values explicitly passed in)
-        params = params || this.get('frameConfig');
+       var params = this.get('frameConfig');
 
         var defaultParams = {};
         Object.keys(this.get('meta.parameters').properties || {}).forEach((key) => {
             defaultParams[key] = this.get(`meta.parameters.properties.${key}.default`);
         });
 
         Object.keys(this.get('meta.data').properties || {}).forEach((key) => {
-            if (this[key] && this[key].isDescriptor) {
```

Comment:
Hmm. Per previous suggestion, I'm actually not in favor of passing the responses to the frame via this mechanism anyway.

The JSON payload we pass to a frame is meant for configuration, not data- putting the previous session responses into a separate variable passed in to the component might be preferred.

Besides not breaking compatibility with other experiments, this also helps us to avoid bugs that could happen if a field in the server stored data happened to match the name of a variable used for configuration. It's a good idea to keep different data in separate buckets if possible.